packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -392,79 +263,168 @@ export default class TraceStatistics extends Component<Props, State> {
+      },
+      {
+        title: 'Min',
+        dataIndex: 'min',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.min},{"type": b.type,"index": b.min}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Max',
+        dataIndex: 'max',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.max},{"type": b.type,"index": b.max}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Total',
+        dataIndex: 'selfTotal',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfTotal},{"type": b.type,"index": b.selfTotal}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Avg',
+        dataIndex: 'selfAvg',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfAvg},{"type": b.type,"index": b.selfAvg}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Min',
+        dataIndex: 'selfMin',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMin},{"type": b.type,"index": b.selfMin}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Max',
+        dataIndex: 'selfMax',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMax},{"type": b.type,"index": b.selfMax}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST in Duration',
+        dataIndex: 'percent',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.percent},{"type": b.type,"index": b.percent}),
+        render: (cell: string) => {
+              return cell+'%'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+    ];
+    let isDetailArray: ITableSpan[] = [];
+    let isNoDetail: ITableSpan[] = [];
+  
+    const findTablesAccToSelectors = (tableValue: ITableSpan[]) => {
+      for (let i = 0; i < tableValue.length; i++) {
+        if (tableValue[i].isDetail) {
+          isDetailArray.push(tableValue[i]);
+        } else {
+          isNoDetail.push(tableValue[i]);
+        }
+      } 
+      for(let i = 0; i < isNoDetail.length; i++){
+          let newArr = isDetailArray.filter((value)=>{return ((value.parentElement) === (isNoDetail[i].name))})
+          newArr = newArr.map((value,index)=>{
+            const x = i.toString()+ index.toString();
+            const keyo = {
+              key: x
+            }
```

Comment:
```suggestion
            const _key = {
              key: `${i}-${index}`
            }
```
Your logic to use `const x = i.toString()+ index.toString();` as a *unique* key won't work when we have `i=1, index=23` and `i=12, index=3`. Let's use a separator like `-` for creating a unique key.

nit: let's use a `_` prefix for variables which are internally used.